impl/src/test/java/org/ehcache/impl/internal/loaderwriter/writebehind/AbstractWriteBehindTestBase.java

```diff
@@ -71,6 +71,7 @@ public void testWriteOrdering() throws Exception {
 
     try (CacheManager cacheManager = managerBuilder().using(cacheLoaderWriterProvider).build(true)) {
       Cache<String, String> testCache = cacheManager.createCache("testWriteOrdering", configurationBuilder()
+        .withLoaderWriter(loaderWriter)
```

Comment:
Thats the way test is written, but ideally there is no concept of default loaderwriter being provided to the cache by the provider. Provider is just a service, but cache needs to be explicitly configured with a loaderwriter, so that provider can provide it.